### PR TITLE
add support find coffeescript file extension for store

### PR DESCRIPTION
**Pull request by mass3ff3ct**
Hello.
I'm added support compilation CoffeScript files (https://github.com/mass3ff3ct/catberry-coffee)
But, StoreFinder search only .js files in catberry_stores.
I added the ability to search for files with the extension .coffee

### CODE REVIEW

File: lib/finders/StoreFinder.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ var path = require('path'),
 	glob = require('glob');
 
 var DEFAULT_STORES_ROOT = 'catberry_stores',
-	STORES_GLOB = '**/*.js';
+	STORES_GLOB = '**/*.@(js|coffee)';
```
Comment (pragmadash) [inline lib/finders/StoreFinder.js]: We have to put this parameter to the [configuration](http://catberry.org/documentation#config) like it's done for componentsGlob parameter. Also, we should keep `STORES_GLOB` as `DEFAULT_STORES_GLOB` in this case.

IMO putting `.coffee` extension in Catberry's code isn't a good practice. Framework should know nothing about its plugins.

**Comment by pragmadash** [comment]
Hi @mass3ff3ct,

Thank you for the pull request. I think we can add `.coffee` extension support but we should do it the other way I've described in code comments.

**Comment by pragmadash** [comment]
Also, don't forget to make changes in the documentation adding a new parameters in the config section.

**Comment by mass3ff3ct** [comment]
@pragmadash, i'm add custom setting storesGlob. But, I see many other merge commits.
I did something wrong?)

**Comment by pragmadash** [comment]
@mass3ff3ct it seems like you merged `master` to `develop`. So, maybe we should create another PR for that? Or you can do `push --force` with the clean branch in this PR.

**Comment by pragmadash** [comment]
Also, take a look at code style error using `npm test`.

**Comment by mass3ff3ct** [comment]
Close.

**Pull request closed without merging**